upmc-enterprises/cda2fhir
# Upmcfhir 331 improve logging

#### What does this PR do?

Logs a message when the parser either encounters a section it doesn't support, or an OID without a URL mapped to it.

Opted not to log when it encontered an unmapped performer role since this requirement goes away in FHIR 4.

#### Related JIRA tickets:
[UPMCFHIR-331](https://jira.amida.com/browse/UPMCFHIR-331)

#### How should this be manually tested?

Run the integration tests and see the results, should have a couple of more messages.

#### Background/Context:

#### New JIRA tickets for clinical review required?
